src/test/java/gov/di_ipv_fraud/pages/FraudPageObject.java

```diff
@@ -214,4 +231,48 @@ public void goToResponse(String validOrInvalid) {
     public void goToVerifiableCredentialsPage() {
         title.getText();
     }
+
+    public void goToEditUserLink() {
+        editUserLink.click();
+    }
+
+    public void addHouseName(String housename) {
+        housenameField.sendKeys(housename);
+    }
+
+    public void clearHouseNumber() {
+        housenumberField.clear();
+    }
+
+    public void addHouseNumber(String housenumber) {
+        housenumberField.sendKeys(housenumber);
+        fraudCRIButton.click();
+    }
+
+    public void userAddressInJsonResponse(String testHouseName) throws JsonProcessingException {
+        String testHouseNumber = "455";
+        String result = JSONPayload.getText();
+        LOGGER.info("result = " + result);
+        ObjectMapper objectMapper = new ObjectMapper();
+        JsonNode jsonNode = objectMapper.readTree(result);
+        JsonNode vcNode = jsonNode.get("vc");
+        JsonNode addressNode = vcNode.get("credentialSubject");
+        JsonNode insideAddress = addressNode.get("address");
+        JsonNode addressContent = insideAddress.get(0);
```

Comment:
I think its worth breaking 254 to 261 into a private method so we can reuse to test other bits of address if we want to

Reply:
@gds-jbentham Have moved this to a private method.